frontend/src/components/Nav/Page/RenderHeader.tsx

```diff
@@ -28,8 +28,11 @@ const actionsToolbarStyle = style({
   borderBottom: '1px solid #d2d2d2'
 });
 
-interface RenderHeaderProps {
+type ReduxProps = {
   isKiosk: boolean;
+}
```

Comment:
I don't think we want both `kiosk` and `isKiosk` in global state.  Is this intentional or maybe an oversight?